upup/pkg/fi/resources.go

```diff
@@ -30,6 +30,11 @@ type Resource interface {
 	Open() (io.Reader, error)
 }
 
+// HasIsReady is implemented by Resources that are derived (and thus may not be ready at comparison time)
+type HasIsReady interface {
```

Comment:
Seems like it might be useful, but I don't see where it's being referenced.

Reply:
Good point - an artifact of trying to resplit the commits.  Removed from this commit.

Reply:
And sent https://github.com/kubernetes/kops/pull/9837